contracts/test/integration/utils/MintRedeemVerification.sol

```diff
@@ -207,6 +205,11 @@ contract MintRedeemVerification {
             ? IVolt(MainnetAddresses.VOLT)
             : IVolt(ArbitrumAddresses.VOLT);
 
+        /// skip Arbitrum minting because minting is paused on Arbitrum due to deprecation
+        if (block.chainid != 1) {
+            revert("successfully minted on all PSMs");
```

Comment:
that's a weird revert error message

Reply:
we use reverts to verify certain operations happened successfully while also rolling back state. This will be migrated to vm.snapshot() foundry functionality in the future.

Reply:
interesting, thanks for the explaination